shared/extensions-marketplace/ai-noise-suppression/reference/web.mdx

```diff
@@ -115,19 +158,34 @@ export interface AIDenoiserExtensionOptions {
 }
 ```
 
-Options for initializing the <Vg k="RTEE_NS" /> extension:
-- `assetsPath`: The host URL of the .wasm and .js files required by the <Vg k="RTEE_NS" /> extension.
+Options for initializing the AI Noise Suppression extension:
+- `assetsPath`: The host URL of the .wasm files required by the AI Noise Suppression extension.
 
+#### AIDenoiserProcessorMode
+
+```typescript
+export enum AIDenoiserProcessorMode {
+  NSNG = "NSNG",
+  STATIONARY_NS = "STATIONARY_NS",
+}
+```
+
+The noise suppression mode:
+
+- `NSNG`: AI noise suppression. This mode suppresses stationary and non-stationary noise described in [Noise types](#type).
+- `STATIONARY_NS`: Stationary noise suppression. This mode suppresses stationary noise only. Agora recommends that you use this mode only when AI noise suppression takes too long.
 
-### Considerations
 
-Currently, the <Vg k="RTEE_NS" /> extension has the following limitations:
+#### AIDenoiserProcessorLevel
 
-- If the sample rate of the input signal is not 16 KHz, the extension downsamples the signal to 16 KHz, removes noise, and resamples the output signal to the original sample rate. This means the audio data above 8 KHz is removed in the output signal.
-- In some unknown scenarios, the extension could cause audio quality to decrease by a certain degree.
-- When multiple people speak at the same time, the audio quality of lowest human voices could be decreased by a certain degree.
-- If only some of the audio tracks on the current web page enable the extension, the audio tracks that do not enable the extension could be affected because the extension turns on AEC and AGC and turns off NS in the browser.
-- While the extension supports Safari later than 14.1, Agora does not recommend using the extension on Safari because of Safari's performance issues.
-- This extension does not support browsers on mobile devices.
+```typescript
```

Comment:
Enumeration values were renamed this morning. See the details in #828:

"[Noise suppression.md](https://github.com/AgoraIO/Docs-Source/files/9950448/Noise.suppression.md)---This doc is updated again because engineering team made an instant change of an API name. Please use [Noise suppression_revised.md](https://github.com/AgoraIO/Docs-Source/files/9958324/Noise.suppression_revised.md) for diff"

Reply:
Done